2025-06-23 14:59:28,304 INFO    MainThread:26088 [wandb_setup.py:_flush():81] Current SDK version is 0.20.1
2025-06-23 14:59:28,305 INFO    MainThread:26088 [wandb_setup.py:_flush():81] Configure stats pid to 26088
2025-06-23 14:59:28,305 INFO    MainThread:26088 [wandb_setup.py:_flush():81] Loading settings from C:\Users\LiamMurphy(DMC)\.config\wandb\settings
2025-06-23 14:59:28,305 INFO    MainThread:26088 [wandb_setup.py:_flush():81] Loading settings from C:\Users\LiamMurphy(DMC)\Documents\Weave\eval\wandb\settings
2025-06-23 14:59:28,305 INFO    MainThread:26088 [wandb_setup.py:_flush():81] Loading settings from environment variables
2025-06-23 14:59:28,306 INFO    MainThread:26088 [wandb_init.py:setup_run_log_directory():703] Logging user logs to C:\Users\LiamMurphy(DMC)\Documents\Weave\eval\wandb\run-20250623_145928-p1i1qmbp\logs\debug.log
2025-06-23 14:59:28,307 INFO    MainThread:26088 [wandb_init.py:setup_run_log_directory():704] Logging internal logs to C:\Users\LiamMurphy(DMC)\Documents\Weave\eval\wandb\run-20250623_145928-p1i1qmbp\logs\debug-internal.log
2025-06-23 14:59:28,307 INFO    MainThread:26088 [wandb_init.py:init():831] calling init triggers
2025-06-23 14:59:28,307 INFO    MainThread:26088 [wandb_init.py:init():836] wandb.init called with sweep_config: {}
config: {'_wandb': {'code_path': 'code/weave_koala/koala_weave_generation.py'}}
2025-06-23 14:59:28,307 INFO    MainThread:26088 [wandb_init.py:init():872] starting backend
2025-06-23 14:59:29,238 INFO    MainThread:26088 [wandb_init.py:init():875] sending inform_init request
2025-06-23 14:59:29,386 INFO    MainThread:26088 [wandb_init.py:init():883] backend started and connected
2025-06-23 14:59:29,390 INFO    MainThread:26088 [wandb_init.py:init():956] updated telemetry
2025-06-23 14:59:29,513 INFO    MainThread:26088 [wandb_init.py:init():980] communicating run to backend with 90.0 second timeout
2025-06-23 14:59:31,586 INFO    MainThread:26088 [wandb_init.py:init():1032] starting run threads in backend
2025-06-23 14:59:32,785 INFO    MainThread:26088 [wandb_run.py:_console_start():2453] atexit reg
2025-06-23 14:59:32,785 INFO    MainThread:26088 [wandb_run.py:_redirect():2301] redirect: wrap_raw
2025-06-23 14:59:32,787 INFO    MainThread:26088 [wandb_run.py:_redirect():2370] Wrapping output streams.
2025-06-23 14:59:32,788 INFO    MainThread:26088 [wandb_run.py:_redirect():2393] Redirects installed.
2025-06-23 14:59:32,792 INFO    MainThread:26088 [wandb_init.py:init():1078] run started, returning control to user process
2025-06-23 15:01:09,410 INFO    MsgRouterThr:26088 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 3 handles.
2025-06-23 15:05:57,452 ERROR   MainThread:26088 [redirect.py:_on_write():664] [no run ID] error in stdout callback
Traceback (most recent call last):
  File "C:\Users\LiamMurphy(DMC)\Documents\Weave\venv\Lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "C:\Users\LiamMurphy(DMC)\Documents\Weave\venv\Lib\site-packages\wandb\sdk\wandb_run.py", line 2374, in <lambda>
    lambda data: self._console_raw_callback("stdout", data),
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\LiamMurphy(DMC)\Documents\Weave\venv\Lib\site-packages\wandb\sdk\wandb_run.py", line 391, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\LiamMurphy(DMC)\Documents\Weave\venv\Lib\site-packages\wandb\sdk\wandb_run.py", line 449, in wrapper_fn
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\LiamMurphy(DMC)\Documents\Weave\venv\Lib\site-packages\wandb\sdk\wandb_run.py", line 1561, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "C:\Users\LiamMurphy(DMC)\Documents\Weave\venv\Lib\site-packages\wandb\sdk\interface\interface.py", line 772, in publish_output_raw
    self._publish_output_raw(o)
  File "C:\Users\LiamMurphy(DMC)\Documents\Weave\venv\Lib\site-packages\wandb\sdk\interface\interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "C:\Users\LiamMurphy(DMC)\Documents\Weave\venv\Lib\site-packages\wandb\sdk\interface\interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "C:\Users\LiamMurphy(DMC)\Documents\Weave\venv\Lib\site-packages\wandb\sdk\lib\sock_client.py", line 174, in send_record_publish
    self.send_server_request(server_req)
  File "C:\Users\LiamMurphy(DMC)\Documents\Weave\venv\Lib\site-packages\wandb\sdk\lib\sock_client.py", line 154, in send_server_request
    self._send_message(msg)
  File "C:\Users\LiamMurphy(DMC)\Documents\Weave\venv\Lib\site-packages\wandb\sdk\lib\sock_client.py", line 151, in _send_message
    self._sendall_with_error_handle(header + data)
  File "C:\Users\LiamMurphy(DMC)\Documents\Weave\venv\Lib\site-packages\wandb\sdk\lib\sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
           ^^^^^^^^^^^^^^^^^^^^^
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
